### DISCUSSION TITLE
‪‏‏‭‮‭‌‬‭‪‏‏‪⁠‌‫‫‍​‬‭‬‪​‭‏‭农场流速分配采用成长率玩法的建议及分析

### DISCUSSION THREAD

**Discussion by yancanlou** [famland/website#125] [Ideas]
建议引入农场成长率分配玩法，现有的分配玩法一直地强化先发优势，形成了垄断式成长，打压新田新农场，后进农场纷纷破产，进入原生农场，将系统生态推入危险之地！趁资金池回落之势，已到必须改变之时！

这里只单纯考察农场成长率100%占比总流速权重的情况。

子农场平均成长率=（本周期末子农场平均共享值-上周期末（本周期始）子农场平均共享值）/上周期末（本周期始）子农场平均共享值
（对于新农场通用就是本周期始或激活时）
子农场流速系数=子农场平均成长率*子农场农田数

子农场流速=子农场流速系数/全部农场的流速系数*总流速

1，子农场成长率为0的发生情况
什么时候子农场成长率为0，
一是，这个农场被放弃了，没有任何人耕种（不在提增量，只管挖提卖）。
二是，农场采取的短期策略，伺机而动：只收取不加增量，等待一个周期尾盘确认有利加时再加。

2，子农场成长率为0会发生什么？
当周期农场流速为0，仅当周期没有流量。分发池无积累，完全靠分发池的库存来供给APY。
第一，收益由APY决定，有收益一样有增量可加，分发池用完也不加？可以空着一直等，等过年来拆都可以！
第二，子农场中各田上肯定都存在一定程度未完全加完增量的储备田。因为这个农场在挖提卖，所以有大把可以加的增量！不管是不是策略都具有突增成长值的暴发力！
第三，加入新的大于农场平均值或退出小于农场平均值的农田，就可以有成长值。

3，子农场的成长率为负值的发生情况
什么时候子农场成长率为负值，
一是，这个农场被放弃了，不只无人耕种，还不断有人退出。
二是，这个农场本周期大于平均共享值的田退出或小于平均共享值的田加入了，引起平均值下降。

4，子农场的成长率为负值会发生什么？
当周期农场流速为负值（可不对分发池做回流处理，防刷时也可）
第一，不对分发池做回流处理，其它农场的总值会大于总流速，会提高引起其它农场流速提高。
第二，对分发池做回流处理，分发池不增反减，农场危机，必引发重视不处理则自然死掉。
如子农场当周期为负，下周期平均值下降后都可以正常成长，拉人或拉田或提增量都是必要的。

5，子农场的成长率为正值的发生情况
一是，这个农场有人耕种，但部分或全部挖提卖时，共享金充足也有增量在加。
二是，这个农场本周期大于平均共享值的田加入或小于平均共享值的田退出了，引起平均值上升。

6，子农场的成长率为正值会发生什么？
当周期的流速为正，分发池在不断积累中，农场活力增加。
第一，收益增加，增加了与其它农场竞争中的增量调用的裕度。
第二，农场成长了，平均共享值的峰值会增加，增加了对分发池的调度APY会上升，也提升了增量与收益空间。

7，博弈的空间与娱乐性（与人斗其乐无穷）；
第一，有了增量可以存着，待尾盘伺机而动，当其它人都加得少或少人加时，可以突击加上。
极端情况，别人都不加，你加个1U（或0.1U）的增量，总流速全部在你的农场，越小时越有这种情况。
规模化后，基本不会发生，如比特币挖矿一样，指望别人矿机都不开，一家独开？
第二，在其它玩家都不停有人加增量情况下，农场增量不足竞争时，一两周期不加就光收益，积累增量空间。
于低落时出击，这样或许比平常占比更高，流速更高。
第三，小农场与大农场可以有一比之力的核心要素在于管理与共识水平高低比赛，产生参与游戏的乐趣。

8，游戏的公平性与成长性
第一，成长率公平分配，是针对自身体量，不论农场大小系数是一样的，100U田收1U,跟500U田收5U，都是1%的成长率。
第二，充分体现了有耕种才有收益的玩法，有新增基本田或野田，就有共享金，就有不影响无损生态的挖提卖。
第三，农场间会欢迎共享值高的农田进入，因为可以提升成长率。
第四，农场间会踢出共享值低的农田，因为也可以提升成长率。
第五，如此情况下，接近的共享值农田的农场其组合稳定成长率，农场间农田的组合会有合理加强共识的趋向。
第六，新人进场可以组合新人（或拉人）组合小田农场，只要肯耕种，一样有流速，不比老玩家差。
第七，因为原生农场的管理无法实现就没有成长率，原生农田内活的田将会减小，玩家建立农场的意愿提高。
第八，原生农场减小，有利于共享农场做大，维护了生态平衡，同时又降低了准拆田队伍对生态的危险影响。
第九，高共享值农场将有极具共识的老玩家农场组成的，占有着APY高比的增量加成，成长率增幅大，管理得好可以让农场进入正螺旋！
APY高→收益率高→成长率高→流速高→分发池高→APY高！
第十，有种树的肥力值高的老玩家，对于增量的系数提升是正向作用的，也将增加成长率。
管理得好也可以进入正螺旋！

**Comment by otto-001** [comment]
还要有个配套，
当基本田发生过撤资的情况下，再次存入资金做市才需要100%手续费。

没有撤资前，不论分多少次存入资金都不收手续费

这样大家就可以100试单，玩明白了，就可以加仓了
共享值只增不减，撤资后再做市收取手续费，也防止刷共享值

**Comment by linshande** [comment]
单田收益率差异由三要素决定：由单田做市资金价值决定，由田系数与肥力值决定，由农场APY决定！

**Reply by linshande** [reply]
抄底农场和新田农场将具有资金价值高
系数由八宝树提升，与肥力值提升
APY由最高平均共享值排名决定

**Comment by linshande** [comment]
所以：
新田有资金净值优势，鼓励建新田。
老田有先发收益打底，鼓励种树！
老田还有肥力值高优势！
老田占有APY排名高优势！

**Reply by linshande** [reply]
整体方向没有大问题，除了小作坊时竞争（规模化后操作竞争可无视），后期提高成长率核心在于提高收益率，提高收益率核心在于收益公式三要素！
贯穿公平，体现差弄，多劳多得，合作共赢，分配均衡！

**Comment by yancanlou** [comment]
为防止刷田：
1，对于高共享值农田空降低平均共享值的农场（高入后，高再出，高再入）
2，对于低共享值田进入高共享值农场（低入后，低再出，低再入）
必须采用负值流速时，农场内分发池倒流回蓄水池防刷，并使得生态自动平衡！！！
还可以从以下方面考虑：
第一，如采用控制进入农场资格，系统增设加入共享农场共享值上限阀值，阀值可取共享农场平均值上浮5%，高于无法加入！
第二，如采用控制进入农场的成本：
高（低）于农场平均共享值的田进入农场，将额外收取系统手续费（可取共享值差额）=丨进入田的共享值-农场平均共享值丨
例：
共享值900田加入平均共享值500农场收取400YL×1（广告折扣50%手续费）。
共享值100田加入平均共享值500农场收取400YL×1（广告折扣50%手续费）。
第三，如采用控制成长率方式，可设农场最高日成长率上限不大于3%，大于取3%。

三种方法供参考。

**Comment by yancanlou** [comment]
一些人说用抢占农场份额不好，其实理解偏差！
用历史（赚钱的增量）数据（赚的已转成基本田或野田）先占，独占哪里好？
加增量时，
人人都想着他人时或只想少赚时，博弈后人人都赚一样多！每块田一天加最少的1U，结果就是人人都大赚！（支少入多）
这就是人人为我，我为人人，利人利己。
人人都想着自己时或都想多赚时，博弈后还是人人都赚一样多！但每块田都加了大额增量，结果就是人人都小赚！（支多入少）
这就是人人为己，损人损己。
这就是社区的共识水平的直接反映，人性反映其中的乐趣。
上次刷田可观一二，路上且远着。

**Comment by yancanlou** [comment]
还有些人定位出错了！天天等着大资本，大资本是看哪里有肉才去哪里的，没有韭菜割不会进去的！
八宝树就是这么玩死的核心要素之一！
善待新田才有出路，那是你的发展之根。
只有草根才能成就草根！
自救成功，生态火爆了！
有利可图，别人才看得上。
大资本从来不会雪中送炭，只有锦上添花与趁火打劫！

**Comment by yancanlou** [comment]
更有的人，总想躺赢？
拼算力的比特币挖矿不能躺赢，为什么？
还是因为人性。
天天投入电费，人工，养护，巨额的设备升级更替，不停扩大，原因就是都想多赚引起的。
那么拼什么能躺着，拼积累记录（增量）可以，先发越积越多，越积越快，你得第一把抓住机会的人，你是吗？
很多人不是，只好观望远离！
这就扼杀了未来，老本吃完，内耗到死！

还有一种是平均分配能躺赢啊，吃大锅饭就是这条路，资本大多吃，资本小少吃，结果大资本来了就等于是控制，控制后什么结果？参考大丰收！

再有一种是每天摇号分配，摇到多少是多少？这手段公平，分配科学吗？而且××社会反对赌博啊！


多劳多得，按劳分配！
这个是社会主义分配的核心原则！
体现个人与团队差异化需求！
把赚的钱给自己造田，以便于加成长值，同时社区成长！（钱还是你的，成长是共享的）
这就是共享精神！
由经营管理与共识博弈，全员参与，起点公平，任何时候利好新田又照顾老田，这才科学合理。

守好主田，小田加共享金，拆旧田灵活，建新田利好，规模化自然有成长动力，这个才是根本之根！
